emojis/emojis.py

```diff
@@ -1,17 +1,23 @@
-import asyncio
 import json
+import os
 from datetime import datetime
 from pytz import timezone
 
+# Get the directory of the current script
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
+# Construct the path to the contributors.json file
+json_file_path = os.path.join(script_dir, 'contributors.json')
```

Comment:
Not sure why you are doing this 
everything is static here or at least i would hope so

Reply:
Was having some problems with the file being read and duplicate file being created. Have added the file path as a constant and am using this instead.